apps/contracts/scripts/mint-lions.ts

```diff
@@ -35,77 +60,36 @@ function getImage(path: string) {
 }
 
 function processLionData(data: LionData) {
-  const {
-    gender,
-    birthday,
-    coalition,
-    prides,
-    unique_features,
-    whisker_right_0,
-    whisker_right_1,
-    whisker_right_2,
-    whisker_left_0,
-    whisker_left_1,
-    whisker_left_2,
-    ear_left,
-    ear_right,
-    face,
-    mouth,
-    id,
-    name,
-  } = data;
-
-  const image = getImage(`${face}`);
-
-  const attributes = [
-    { trait_type: 'id', value: id },
-    { trait_type: 'gender', value: gender },
-    { trait_type: 'birthday', value: birthday },
-    { trait_type: 'coalition', value: coalition },
-    {
-      trait_type: 'prides',
-      value: prides,
-    },
-    { trait_type: 'uniqueFeatures', value: unique_features },
-    {
-      trait_type: 'whiskersRightImage0',
-      value: whisker_right_0 ? getImage(`${whisker_right_0}`) : '',
-    },
-    {
-      trait_type: 'whiskersRightImage1',
-      value: whisker_right_1 ? getImage(`${whisker_right_1}`) : '',
-    },
-    {
-      trait_type: 'whiskersRightImage2',
-      value: whisker_right_2 ? getImage(`${whisker_right_2}`) : '',
-    },
-    {
-      trait_type: 'whiskersLeftImage0',
-      value: whisker_left_0 ? getImage(`${whisker_left_0}`) : '',
-    },
-    {
-      trait_type: 'whiskersLeftImage1',
-      value: whisker_left_1 ? getImage(`${whisker_left_1}`) : '',
-    },
-    {
-      trait_type: 'whiskersLeftImage2',
-      value: whisker_left_2 ? getImage(`${whisker_left_2}`) : '',
-    },
-    {
-      trait_type: 'earRight',
-      value: ear_right ? getImage(`${ear_right}`) : '',
-    },
-    {
-      trait_type: 'earLeft',
-      value: ear_left ? getImage(`${ear_left}`) : '',
-    },
-    { trait_type: 'face', value: image },
-    { trait_type: 'mouth', value: mouth ? getImage(`${mouth}`) : '' },
-  ];
+  const image = getImage(`${data.face}`);
```

Comment:
it needs to be profile here.

Reply:
Because it is the terminology that KWT uses

Reply:
Yes, will change this in next version.